frontend-opal/components/Header/Header.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Button, View, TextInput, Image } from 'react-native';
+import styles from './styles';
+import ListIcon from '../../assets/list-icon.png';
+import SearchIcon from '../../assets/search-button.png';
+
+const Header = () => {
+  const narTitles = [
+    { id: 1, key: 'PORTFOLI' },
+    { id: 2, key: 'WATCHLIST' }
+  ];
+
+  const handlePress = key => key === 'PORTFOLIO';
+
+  const renderItems = titles =>
+    titles.map(title => (
+      <View
+        key={title.id}
+        style={[styles.narTitle, handlePress(title.key) && styles.buttonPressed]}
+      >
+        <Button title={title.key} color="#e17055" />
+      </View>
+    ));
+
+  return (
+    <View>
+      <View style={styles.search}>
+        <Image style={styles.listIcon} source={ListIcon} />
+        <TextInput style={styles.input} />
+        <Image style={styles.searchButton} source={SearchIcon} />
+      </View>
+
+      <View style={styles.navBars}>{renderItems(narTitles)}</View>
```

Comment:
@Yuan-EPAM you can convert this line into a separate component, or convert the single element into a component, but calling a function `renderItems` is exactly that (it has a prop: `titles` and it returns a React element)